src/query/management/tests/it/warehouse.rs

```diff
@@ -314,7 +314,17 @@ async fn test_successfully_create_system_managed_warehouse() -> Result<()> {
         SelectedNode::Random(None),
     ]);
 
-    create_warehouse.await?;
+    let cw = create_warehouse.await?;
+    if let WarehouseInfo::SystemManaged(sw) = cw {
```

Comment:
Need to assert that it is not self-managed.

Reply:
Done